.github/actions/create-jira-issues-for-community-activities/action.yml

```diff
@@ -32,7 +32,7 @@ runs:
         PR_TITLE: ${{ github.event.pull_request.title }}
       shell: pwsh
       run: |
-        $output = "${{ github.event.pull_request }}" -eq "" ? "False" : (Confirm-PullRequestTitle "$($Env:PR_TITLE)") -or $Env:GITHUB_HEAD_REF -Like "*issue*"
+        $output = "${{ github.event.pull_request }}" -eq "" ? "False" : (Confirm-PullRequestTitle "$Env:PR_TITLE") -or $Env:GITHUB_HEAD_REF -Like "*issue*"
```

Comment:
Do we actually need quotes around `$Env:PR_TITLE`, given that the variable is the sole content of that string parameter?

Reply:
In case it contains spaces, they might be necessary.

I just removed the superfluous extra `$()` wrapper because this was the only place in LGHA where we had that.